packages/core/src/browser/consoleObservable.ts

```diff
@@ -2,41 +2,29 @@ import { callMonitored } from '../domain/internalMonitoring'
 import { computeStackTrace } from '../domain/tracekit'
 import { createHandlingStack, ErrorSource, formatErrorMessage, toStackTraceString } from '../tools/error'
 import { mergeObservables, Observable } from '../tools/observable'
-import type { ClocksState } from '../tools/timeUtils'
-import { clocksNow } from '../tools/timeUtils'
 import { find, jsonStringify } from '../tools/utils'
 
-const StatusType = {
+export const ConsoleApiName = {
+  log: 'log',
   debug: 'debug',
   info: 'info',
   warn: 'warn',
   error: 'error',
 } as const
 
-type StatusType = typeof StatusType[keyof typeof StatusType]
-
-type ApiNameType = 'log' | 'debug' | 'info' | 'warn' | 'error'
-
-const LogStatusForApi = {
-  log: StatusType.info,
-  debug: StatusType.debug,
-  info: StatusType.info,
-  warn: StatusType.warn,
-  error: StatusType.error,
-}
+type ConsoleApiNameType = typeof ConsoleApiName[keyof typeof ConsoleApiName]
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏It's fine to use an enum here (this is not public facing). It would be a bit simpler.